code/mob/living/critter/flockcritter_parent.dm

```diff
@@ -62,7 +62,6 @@ TYPEINFO(/mob/living/critter/flock)
 /mob/living/critter/flock/New(var/atom/L, var/datum/flock/F=null)
 	..()
 	remove_lifeprocess(/datum/lifeprocess/radiation)
-	src.material.setProperty("reflective", 5)
```

Comment:
Does this get done somewhere else? because this seems like a (very minor) balance change if the stats are actually changing

Reply:
yeah, this was discussed in #flockmind, and we essentially came to the conclusion "why do flock drones even have material". Leah was gonna remove it.